nmostesting/suites/IS1301Test.py

```diff
@@ -0,0 +1,316 @@
+# Copyright (C) 2023 Advanced Media Workflow Association
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+The script implements the IS-13 test suite as specified by the nmos-resource-labelling workgroup.
+At the end of the test, the initial state of the tested unit is supposed to be restored but this
+cannot be garanteed.
+
+In addition to the basic annotation API tests, this suite includes the test sequence:
+For each resource type (self, devices, senders, receivers):
+    For each annotable object type (label, description, tags):
+        - Read initial value
+            - store
+        - Reset default value by sending null
+            - check value + timestamp + is-14/value + is-04 timestamp
+            - store
+        - Write max-length
+            - check value + timestamp + is-14/value + is-04 timestamp
+        - Write >max-length
+            - check value + timestamp + is-14/value + is-04 timestamp
+        - Reset default value again
+            - check value + timestamp + is-14/value + is-04 timestamp
+            - compare with 1st reset
+        - Restore initial value
+"""
+
+from ..GenericTest import GenericTest, NMOSTestException
+from ..TestHelper import compare_json
+
+from ..import TestHelper
+import re
+import copy
+import time
+
+ANNOTATION_API_KEY = "annotation"
+NODE_API_KEY = "node"
+
+RESOURCES = ["self", "devices", "senders", "receivers"]
+OBJECTS = ["label", "description", "tags"]
+
+# const for label&description-related tests
+STRING_OVER_MAX_VALUE = ''.join(['X' for i in range(100)])
+STRING_MAX_VALUE = STRING_OVER_MAX_VALUE[:64]  # this is the max length tolerated

```

Comment:
The parameter naming here gives the first impression these are number values (cf. INT_MAX).

Perhaps add `LENGTH` in as well `STRING_MAX_LENGTH_VALUE`? A little verbose but less confusing.

Similarly `STRING_OVER_MAX_VALUE` could be `STRING_LENGTH_OVER_MAX_VALUE` or `STRING_OVER_MAX_LENGTH_VALUE`